src/include/miopen/solver/ck_utility_common.hpp

```diff
@@ -71,7 +71,8 @@ static inline bool is_ck_supported_hardware(const Handle& handle)
 static inline bool is_ck_whitelist(const std::string& device_name)
 {
     return (StartsWith(device_name, "gfx908") || StartsWith(device_name, "gfx90a") ||
-            StartsWith(device_name, "gfx942") || StartsWith(device_name, "gfx950"));
+            StartsWith(device_name, "gfx942") || StartsWith(device_name, "gfx950") ||
+            StartsWith(device_name, "gfx11") );
```

Comment:
This affects the applicability of several solvers.

Reply:
I understand that, but I noticed that MIOpen calls CK's instance factory, which contains logic to determine whether a suitable op is available on a specific gfx. If no op is available, `IsCKApplicable` returns `false`.

Reply:
We only add a device to the whitelist if CK has full support. Currently, CK only enabled partial support on Navi31. Therefore, add this device to the whitelist might cause some issues. I will implement a workaround to enable the partial support before CK has full support on Navi31.